setup.py

```diff
@@ -68,6 +69,7 @@ def run_setup():
         pycryptodome]
     cwl_reqs = [
         cwltool,
+        cwltest,
```

Comment:
Please don't; regular `toil-cwl-runner` users don't need cwltest. How about as another extra named `cwl-dev`?

Reply:
@mr-c +1.  Though all dev reqs as `dev` might be good too (maybe getting rid of `make prepare`?).

Reply:
Oh, we already have `requirements-dev.txt` which is how we get it in the first place. I thought `cwltool` was bringing it along. I will use that mechanism instead.